requirements.txt

```diff
@@ -1,9 +1,10 @@
 boto3>=1.4.3
 botocore>=1.4.93
 hvac>=0.8.1
-PyYaml
+PyYaml>=5.3
```

Comment:
Let's use `>=5.2` since that's known to work and currently `awscli` specifies `<5.3`.